app/backend/src/couchers/utils.py

```diff
@@ -49,11 +50,41 @@ def least_current_date():
     return datetime.now(timezone(timedelta(hours=-12))).strftime("%Y-%m-%d")
 
 
+# Note: be very careful with ordering of lat/lng!
+# In a lot of cases they come as (lng, lat), but us humans tend to use them from GPS as (lat, lng)...
+# When entering as EPSG4326, we also need it in (lng, lat)
+
+
 def create_coordinate(lat, lng):
     """
     Creates a WKT point from a (lat, lng) tuple in EPSG4326 coordinate system (normal GPS-coordinates)
     """
-    return func.ST_SetSRID(func.ST_MakePoint(lng, lat), 4326)
+    return from_shape(Point(lng, lat), srid=4326)
+
+
+def create_polygon_lat_lng(points):
+    """
+    Creates a EPSG4326 WKT polygon from a list of (lat, lng) tuples
+    """
+    return from_shape(Polygon([(lng, lat) for (lat, lng) in points]), srid=4326)
```

Comment:
This reversal is gross but I guess nothing to be done

Reply:
Yeah :/, this function can be avoided most of the time though